sdk/include/psdk/imm.h

```diff
@@ -25,116 +13,118 @@
 extern "C" {
 #endif
 
-#ifdef __REACTOS__
+#if (WINVER >= 0x040A)
 DECLARE_HANDLE(HIMC);
 DECLARE_HANDLE(HIMCC);
-typedef HKL *LPHKL;
 #else
-typedef HANDLE HIMC;
-typedef HANDLE HIMCC;
+typedef DWORD HIMC;
+typedef DWORD HIMCC;
 #endif
 
+typedef HKL FAR *LPHKL;
+typedef UINT FAR *LPUINT;
+
 typedef struct tagREGISTERWORDA {
     LPSTR lpReading;
     LPSTR lpWord;
-} REGISTERWORDA, *PREGISTERWORDA, *NPREGISTERWORDA, *LPREGISTERWORDA;
+} REGISTERWORDA, *PREGISTERWORDA, NEAR *NPREGISTERWORDA, FAR *LPREGISTERWORDA;
```

Comment:
Do we really want _that_ level of compatibility? (since I'm not sure this header will ever be used for compiling 16-bit windows code)

Reply:
Nope in reality, but I keep this code.